### PR TITLE
Replace demisto to Cortex XSOAR

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Replace demisto to Cortex XSOAR

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Malwarebytes/Integrations/Malwarebytes/README.md [hunk 1]
```diff
@@ -38,20 +38,20 @@ This integration collects your E-mail and Company Name for usage analytics of Ma
...
 ---
-The fetch incidents command is the function that Demisto calls every minute to import new incidents and is triggered by the "Fetches incidents" parameter in the integration configuration. 
+The fetch incidents command is the function that Cortex XSOAR calls every minute to import new incidents and is triggered by the "Fetches incidents" parameter in the integration configuration. 
 
 What kind of objects/entities the integration should fetch (events/alerts/incidents/cases/tickets/etc)?
 
-1.	Create a Demisto incident upon a Malwarebytes Real-time protection detections.
-2.	 Create a Demisto incident upon Malwarebytes Suspicious Acitivty detections.
+1.	Create a Cortex XSOAR incident upon a Malwarebytes Real-time protection detections.
+2.	 Create a Cortex XSOAR incident upon Malwarebytes Suspicious Acitivty detections.
```
Comment (yaakovi) [inline Packs/Malwarebytes/Integrations/Malwarebytes/README.md:46]:
```suggestion
2.	Create a Cortex XSOAR incident upon a Malwarebytes Suspicious Activity detections.
```

File: Packs/MongoDB/.pack-ignore [hunk 2]
```diff
@@ -1 +1,4 @@
-https://docs.mongodb.com
\ No newline at end of file
+https://docs.mongodb.com
```
Comment (yaakovi) [inline Packs/MongoDB/.pack-ignore:1]: should this be in the `.secret-ignore` file? please remove from here

File: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_2_15.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Microsoft Defender for Endpoint
+- Fixed an issue where the pagination did not work properly in the ***microsoft-atp-indicator-list*** command. 
+- Updated the Docker image to: *demisto/crypto:1.0.0.19032*.
```
Comment (yaakovi) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_2_15.md:5]: ???

File: Packs/Malwarebytes/Integrations/Malwarebytes/README.md (hunk 1 above)
Comment (yaakovpraisler) [reply Packs/Malwarebytes/Integrations/Malwarebytes/README.md:46]: done

File: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_2_15.md (hunk 3 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_2_15.md:5]: done

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection_description.md [hunk 4]
```diff
@@ -8,4 +8,4 @@ There are 2 application authentication methods available:
 Depending on the authentication method that you use, the integration parameters might change.
 
 To allow us access to Microsoft Defender Advanced Threat Protection, an admin has to approve our app using an admin consent flow, by clicking on the following [link](https://oproxy.demisto.ninja/ms-defender-atp).
-After authorizing the Demisto app, you will get an ID, Token, and Key, which should be inserted in the integration instance configuration's corresponding fields.
+After authorizing the Cortex XSOAR app, you will get an ID, Token, and Key, which should be inserted in the integration instance configuration's corresponding fields.
```
Comment (roysagi) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection_description.md:11]: revert this

File: Packs/MicrosoftGraphCalendar/Integrations/MicrosoftGraphCalendar/MicrosoftGraphCalendar_description.md [hunk 5]
```diff
@@ -7,5 +7,5 @@ There are 2 application authentication methods available:
 
 Depending on the authentication method that you use, the integration parameters might change.
 
-To allow access to Microsoft Graph Calendar, an administrator has to approve the Demisto app using an admin consent flow, by clicking on the following [link](https://oproxy.demisto.ninja/ms-graph-calendar).
-After authorizing the Demisto app, you will get an ID, Token, and Key, which needs to be added to the integration instance configuration's corresponding fields.
\ No newline at end of file
+To allow access to Microsoft Graph Calendar, an administrator has to approve the Cortex XSOAR app using an admin consent flow, by clicking on the following [link](https://oproxy.demisto.ninja/ms-graph-calendar).
+After authorizing the Cortex XSOAR app, you will get an ID, Token, and Key, which needs to be added to the integration instance configuration's corresponding fields.
```
Comment (roysagi) [inline Packs/MicrosoftGraphCalendar/Integrations/MicrosoftGraphCalendar/MicrosoftGraphCalendar_description.md:11]: revert these changes

File: Packs/MicrosoftGraphDeviceManagement/Integrations/MicrosoftGraphDeviceManagement/MicrosoftGraphDeviceManagement_description.md [hunk 6]
```diff
@@ -8,4 +8,4 @@ There are 2 application authentication methods available:
 Depending on the authentication method that you use, the integration parameters might change.
 
 To allow us access to Microsoft Graph Device Management, an admin has to approve our app using an admin consent flow, by clicking on the following [link](https://oproxy.demisto.ninja/ms-graph-device-management).
-After authorizing the Demisto app, you will get an ID, Token, and Key, which should be inserted in the integration instance configuration's corresponding fields.
\ No newline at end of file
+After authorizing the Cortex XSOAR app, you will get an ID, Token, and Key, which should be inserted in the integration instance configuration's corresponding fields.
```
Comment (roysagi) [inline Packs/MicrosoftGraphDeviceManagement/Integrations/MicrosoftGraphDeviceManagement/MicrosoftGraphDeviceManagement_description.md]: revert this change

File: Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles_description.md [hunk 7]
```diff
@@ -7,5 +7,5 @@ There are 2 application authentication methods available:
 
 Depending on the authentication method that you use, the integration parameters might change.
 
-To allow access to Microsoft Graph Files, an administrator has to approve the Demisto app using the [admin consent flow](https://oproxy.demisto.ninja/ms-graph-files).
-After authorizing the Demisto app, you receive an ID, Token, and Key, all of which need to be entered to the corresponding fields when configuring the integration instance.
+To allow access to Microsoft Graph Files, an administrator has to approve the Cortex XSOAR app using the [admin consent flow](https://oproxy.demisto.ninja/ms-graph-files).
+After authorizing the Cortex XSOAR app, you receive an ID, Token, and Key, all of which need to be entered to the corresponding fields when configuring the integration instance.
```
Comment (roysagi) [inline Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles_description.md:11]: revert these changes

File: Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups_description.md [hunk 8]
```diff
@@ -7,6 +7,6 @@ There are 2 application authentication methods available:
 
 Depending on the authentication method that you use, the integration parameters might change.
 
-To allow access to Microsoft Graph Groups, an administrator has to approve the Demisto app using an admin consent flow, by clicking on the following [link](https://oproxy.demisto.ninja/ms-graph-groups).
-After authorizing the Demisto app, you will get an ID, Token, and Key, which needs to be added to the integration instance configuration's corresponding fields.
+To allow access to Microsoft Graph Groups, an administrator has to approve the Cortex XSOAR app using an admin consent flow, by clicking on the following [link](https://oproxy.demisto.ninja/ms-graph-groups).
+After authorizing the Cortex XSOAR app, you will get an ID, Token, and Key, which needs to be added to the integration instance configuration's corresponding fields.
```
Comment (roysagi) [inline Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups_description.md:11]: revert these changes

File: Packs/MicrosoftGraphListener/Integrations/MicrosoftGraphListener/MicrosoftGraphListener_description.md [hunk 9]
```diff
@@ -8,4 +8,4 @@ There are 2 application authentication methods available:
 Depending on the authentication method that you use, the integration parameters might change.
 
 In order to use Cortex XSOAR application and allow us access to Microsoft Graph Mail Single User, you need to approve our app, by clicking on the following [link](https://oproxy.demisto.ninja/ms-graph-mail-listener).
-After authorizing the Demisto app, you will get an ID, Token, and Key, which should be inserted in the integration instance configuration's corresponding fields.
\ No newline at end of file
+After authorizing the Cortex XSOAR app, you will get an ID, Token, and Key, which should be inserted in the integration instance configuration's corresponding fields.
```
Comment (roysagi) [inline Packs/MicrosoftGraphListener/Integrations/MicrosoftGraphListener/MicrosoftGraphListener_description.md]: revert this change

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail_description.md [hunk 10]
```diff
@@ -8,7 +8,7 @@ There are 2 application authentication methods available:
 Depending on the authentication method that you use, the integration parameters might change.
 
 In order to use Cortex XSOAR application and allow us access to Microsoft Graph Mail, an admin has to approve our app using an admin consent flow, by clicking on the following [link](https://oproxy.demisto.ninja/ms-graph-mail).
-After authorizing the Demisto app, you will get an ID, Token, and Key, which should be inserted in the integration instance configuration's corresponding fields.
+After authorizing the Cortex XSOAR app, you will get an ID, Token, and Key, which should be inserted in the integration instance configuration's corresponding fields.
```
Comment (roysagi) [inline Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail_description.md:11]: revert this change

File: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity_description.md [hunk 11]
```diff
@@ -8,4 +8,4 @@ There are 2 application authentication methods available:
 Depending on the authentication method that you use, the integration parameters might change.
 
 To allow us access to Microsoft Graph Security, an admin has to approve our app using an admin consent flow, by clicking on the following [link](https://oproxy.demisto.ninja/ms-graph-security).
-After authorizing the Demisto app, you will get an ID, Token, and Key, which should be inserted in the integration instance configuration's corresponding fields.
\ No newline at end of file
+After authorizing the Cortex XSOAR app, you will get an ID, Token, and Key, which should be inserted in the integration instance configuration's corresponding fields.
```
Comment (roysagi) [inline Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity_description.md]: revert this change

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 12]
```diff
@@ -85,14 +85,14 @@ All HTTPS traffic that will hit the selected messaging endpoint will be directed
...
-Before you can create an instance of the Microsoft Teams integration in Demisto, you need to complete the following procedures.
+Before you can create an instance of the Microsoft Teams integration in Cortex XSOAR, you need to complete the following procedures.
 
 1. [Create the Demisto Bot in Microsoft Teams](#create-the-demisto-bot-in-microsoft-teams)
 2. [Grant the Demisto Bot Permissions in Microsoft Graph](#grant-the-demisto-bot-permissions-in-microsoft-graph)
 3. [Configure Microsoft Teams on Demisto](#configure-microsoft-teams-on-demisto)
 4. [Add the Demisto Bot to a Team](#add-the-demisto-bot-to-a-team)
 
-### Create the Demisto Bot in Microsoft Teams
+### Create the Cortex XSOAR Bot in Microsoft Teams
```
Comment (roysagi) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:95]: revert this change

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 13]
```diff
@@ -102,31 +102,31 @@ Before you can create an instance of the Microsoft Teams integration in Demisto,
 6. Click the **Manifest editor** tab.
 7. Click the **Import an existing app** button, and select the ZIP file that you downloaded.
 8. Click the app widget, and in the **Identification** section, click the **Generate** button to generate a unique App ID.  The following parameters are automatically populated in the ZIP file, use this information for reference.
-  - **Short name**: Demisto Bot
-  - **App ID**: the App ID for configuring in Demisto.
+  - **Short name**: Cortex XSOAR Bot
```
Comment (roysagi) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:105]: revert this change

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 14]
```diff
@@ -102,31 +102,31 @@ Before you can create an instance of the Microsoft Teams integration in Demisto,
...
+  - **App ID**: the App ID for configuring in Cortex XSOAR.
   - **Package name**: desmisto.bot (this is a unique identifier for the app in the Store)
   - **Version**: 1.0.0 (this is a unique identifier for the app in the Store)
-  - **Short description**: Mechanism for mirroring between Demisto and Microsoft Teams.
-  - **Long description**: Demisto Bot is the mechanism that enables messaging team members and channels, executing Demisto commands directly from Teams, and mirroring investigation data between Demisto and Microsoft Teams
+  - **Short description**: Mechanism for mirroring between Cortex XSOAR and Microsoft Teams.
+  - **Long description**: Cortex XSOAR Bot is the mechanism that enables messaging team members and channels, executing Cortex XSOAR commands directly from Teams, and mirroring investigation data between Cortex XSOAR and Microsoft Teams
```
Comment (roysagi) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:110]:
```suggestion
  - **Long description**: Demisto Bot is the mechanism that enables messaging team members and channels, executing Cortex XSOAR commands directly from Teams, and mirroring investigation data between Cortex XSOAR and Microsoft Teams
```

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 15]
```diff
@@ -102,31 +102,31 @@ Before you can create an instance of the Microsoft Teams integration in Demisto,
...
 
 9. From the left-side navigation pane, under Capabilities, click **Bots > Set up**.
 10. Configure the settings under the **Scope** section, and click **Create bot**.
-  - In the **Name** field, enter *Demisto Bot*.
+  - In the **Name** field, enter *Cortex XSOAR Bot*.
```
Comment (roysagi) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:114]: revert this change

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 16]
```diff
@@ -102,31 +102,31 @@ Before you can create an instance of the Microsoft Teams integration in Demisto,
...
   - In the **Scope** section, select the following checkboxes: `Personal`, `Team`, and `Group Chat`. 
 
-11. Record the **Bot ID**, which you will need when configuring the integration in Demisto.
+11. Record the **Bot ID**, which you will need when configuring the integration in Cortex XSOAR.
 ![image](https://raw.githubusercontent.com/demisto/content/b222375925eb13feaaa28cd8b1c814b4d212f2e4/Integrations/MicrosoftTeams/doc_files/MSTeams-BotID.png)
-12. Click **Generate new password**. Record the password, which you will need when configuring the integration in Demisto.
-13. In the **Messaging endpoints** section, enter the URL to which messages will be sent (to the Demisto Bot).
+12. Click **Generate new password**. Record the password, which you will need when configuring the integration in Cortex XSOAR.
+13. In the **Messaging endpoints** section, enter the URL to which messages will be sent (to the Cortex XSOAR Bot).
```
Comment (roysagi) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:120]: revert this change

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 17]
```diff
@@ -102,31 +102,31 @@ Before you can create an instance of the Microsoft Teams integration in Demisto,
...
   - To enable calling capabilities on the Bot enter the same URL to the **Calling endpoints** section.
 14. From the left-side navigation pane, under Finish, click **Test and distribute**.
 15. To download the new bot file, which now includes App Details, click **Download**.
 16. Navigate to Store, and click **Upload a custom app > Upload for ORGANIZATION-NAME**, and select the ZIP file you downloaded.
 
-### Grant the Demisto Bot Permissions in Microsoft Graph
+### Grant the Cortex XSOAR Bot Permissions in Microsoft Graph
```
Comment (roysagi) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:126]: revert this change

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 18]
```diff
@@ -102,31 +102,31 @@ Before you can create an instance of the Microsoft Teams integration in Demisto,
...
 
 1. Go to your Microsoft Azure portal, and from the left navigation pane select **Azure Active Directory > App registrations**.
-2. Search for and click **Demisto Bot**.
+2. Search for and click **Cortex XSOAR Bot**.
```
Comment (roysagi) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:129]: revert this change

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 19]
```diff
@@ -135,12 +135,12 @@ Before you can create an instance of the Microsoft Teams integration in Demisto,
   - Calls.InitiateGroupCall.All
   - OnlineMeetings.ReadWrite.All
 
-5. Verify that all permissions were added, and click **Grant admin consent for Demisto**.
+5. Verify that all permissions were added, and click **Grant admin consent for Cortex XSOAR**.
```
Comment (roysagi) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:138]: revert this change

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 20]
```diff
@@ -165,18 +165,18 @@ Before you can create an instance of the Microsoft Teams integration in Demisto,
 
 4. Click **Test** to validate the URLs, token, and connection.
 
-### Add the Demisto Bot to a Team
+### Add the Cortex XSOAR Bot to a Team
```
Comment (roysagi) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:168]: revert this change

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 21]
```diff
@@ -165,18 +165,18 @@ Before you can create an instance of the Microsoft Teams integration in Demisto,
...
 
   - Note: the following need to be done after configuring the integration on Cortex XSOAR (the previous step).
   
 1. In Microsoft Teams, access the Store.
-2. Search for **Demisto Bot** and click the Demisto Bot widget.
+2. Search for **Cortex XSOAR Bot** and click the Cortex XSOAR Bot widget.
```
Comment (roysagi) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:173]: revert this change

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 22]
```diff
@@ -231,9 +231,9 @@ Mirrors the Demisto investigation to the specified Microsoft Teams channel.
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
 | mirror_type | The mirroring type. Can be "all", which mirrors everything, "chat", which mirrors only chats (not commands), or "none", which stops all mirroring. | Optional | 
-| autoclose | Whether to auto-close the channel when the incident is closed in Demisto. If "true", the channel will be auto-closed. Default is "true". | Optional | 
-| direction | The mirroring direction. Can be "FromDemisto", "ToDemisto", or "Both". | Optional | 
-| team | The team in which to mirror the Demisto investigation. If not specified, the default team configured in the integration parameters will be used. | Optional | 
+| autoclose | Whether to auto-close the channel when the incident is closed in Cortex XSOAR. If "true", the channel will be auto-closed. Default is "true". | Optional | 
+| direction | The mirroring direction. Can be "FromXSOAR", "ToXSOAR", or "Both". | Optional |
```
Comment (roysagi) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:235]: revert this change

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 23]
```diff
@@ -457,7 +457,7 @@ You can run Cortex XSOAR commands, according to the user permissions, from Micro
 
 Note: Like every message in a mirrored channel, in order for it to be passed to the bot, the bot must be mentioned.
 
-For example, in order to check the reputation of the IP address 8.8.8.8, run the following: `@Demisto Bot !ip ip=8.8.8.8`
+For example, in order to check the reputation of the IP address 8.8.8.8, run the following: `@XSOAR Bot !ip ip=8.8.8.8`
```
Comment (roysagi) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:460]: revert this change

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 24]
```diff
@@ -504,7 +504,6 @@ You can send the message `help` in order to see the supported commands:
 
    Make sure to remove the bot from the team before clearing the integration cache, and add it back after done.
 
-
-## Download Demisto Bot
+## Download Cortex XSOAR Bot
```
Comment (roysagi) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:507]: revert this change

**Review by roysagi — CHANGES_REQUESTED** [review]

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection_description.md (hunk 4 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection_description.md:11]: done.

File: Packs/MicrosoftGraphCalendar/Integrations/MicrosoftGraphCalendar/MicrosoftGraphCalendar_description.md (hunk 5 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftGraphCalendar/Integrations/MicrosoftGraphCalendar/MicrosoftGraphCalendar_description.md:11]: done.

File: Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles_description.md (hunk 7 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles_description.md:11]: done.

File: Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups_description.md (hunk 8 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups_description.md:11]: done.

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail_description.md (hunk 10 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail_description.md:11]: done.

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md (hunk 24 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:507]: done.

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md (hunk 23 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:460]: done.

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md (hunk 22 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:235]: done.

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md (hunk 21 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:173]: done.

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md (hunk 20 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:168]: done.

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md (hunk 19 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:138]: done.

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md (hunk 18 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:129]: done.

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md (hunk 17 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:126]: done.

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md (hunk 16 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:120]: done.

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md (hunk 15 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:114]: done.

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md (hunk 14 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:110]: done.

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md (hunk 13 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:105]: done.

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md (hunk 12 above)
Comment (yaakovpraisler) [reply Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:95]: done.

**Review by roysagi — APPROVED** [review]

**Pull request merged**